src/main/slice/omero/model/ElectricPotential.ice

```diff
@@ -32,15 +32,15 @@ module omero {
        * an {@link omero.model.IObject} implementation and as such does
        * not have an ID value. Instead, the entire object is embedded
        * into the containing class, so that the value and unit rows
-       * can be found on the table itself (e.g. detectorSettings.voltage
-       * and detectorSettings.voltageUnit).
-       */
+       * can be found on the table itself (e.g. detectorSettings.volatage
```

Comment:
Seems a strange change.

Reply:
Agreed. Trying to figure out where this came from.

Reply:
Grepping through `ome-units` I don't see where it was generated, but rolling back.